### PR TITLE
Feature/events list

### PR DESCRIPTION
**Pull request by wcjr**

### CODE REVIEW

File: src/components/Nav.svelte [hunk 1]
```diff
@@ -57,11 +57,17 @@ $: totalNumUnreads = $unreads.reduce((sum, { count }) => sum + count, 0)
         {/if}
 
         <li class="nav-item">
-          <a href="/#/requests" class="nav-link" class:active={$location === '/requests'}>
+          <a href="/#/requests" class="nav-link" class:active={$location.startsWith('/requests')}>
```
Comment (forevermatt) [inline src/components/Nav.svelte]: Nice catch!

File: src/data/gqlQueries.js [hunk 2]
```diff
@@ -167,6 +167,26 @@ export async function markMessagesAsRead(threadId) {
   return response.setThreadLastViewedAt || {}
 }
 
+export async function getEvents() {
+  const response = await gql(`{
+    meetings {
```
Comment (forevermatt) [inline src/data/gqlQueries.js]: Do you want to extra the meeting fields to a separate constant, and revise this to `meetings { ${meetingFields} }` like we do for things like posts' fields?

File: src/components/Nav.svelte (hunk 1 above)
Comment (wcjr) [reply src/components/Nav.svelte]: Thanks, I should've done it on `develop` though, sorry about that.

File: src/data/gqlQueries.js (hunk 2 above)
Comment (wcjr) [reply src/data/gqlQueries.js]: yep, as soon as we need it in another query I will.

**Comment by wcjr** [comment]
I know you're sick today @forevermatt .  I need these changes in the new invitations feature so I'm going to go ahead and merge but can always come back and make any updates you're not satisfied with.

**Pull request merged**